src/main/java/frc/robot/subsystems/intake/IntakeOpenerSS.java

```diff
@@ -0,0 +1,58 @@
+package frc.robot.subsystems.intake;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import frc.robot.components.TrigonTalonSRX;
+import frc.robot.constants.RobotConstants.IntakeOpenerConstants;
+import frc.robot.subsystems.OverridableSubsystem;
+import frc.robot.utilities.EncoderConversions;
+
+public class IntakeOpenerSS extends OverridableSubsystem {
+    private final TrigonTalonSRX motor;
+
+    public IntakeOpenerSS() {
```

Comment:
add a reset encoder that sets selected sensor position to 0 and call it in the constructor